libfwupdplugin/fu-bluez-device.c

```diff
@@ -410,6 +416,59 @@ fu_bluez_device_read_battery_interface(FuBluezDevice *self,
 	return TRUE;
 }
 
+/* see https://www.bluetooth.com/specifications/dis-1-2/ spec */
+static gboolean
+fu_bluez_device_parse_device_information_service(FuBluezDevice *self, GError **error)
+{
+	g_autofree gchar *model_number = NULL;
+	g_autofree gchar *serial_number = NULL;
+	g_autofree gchar *fw_revision = NULL;
+	g_autofree gchar *manufacturer = NULL;
+
+	model_number = fu_bluez_device_read_string(self, DI_MODEL_NUMBER_UUID, NULL);
+	if (model_number) {
+		fu_device_add_instance_str(FU_DEVICE(self), "MODEL", model_number);
```

Comment:
`DEV` perhaps?

Reply:
again not sure. might be.
please check naming [DI specification](https://www.bluetooth.com/wp-content/uploads/Files/Specification/HTML/DIS_v1.2/out/en/index-en.html#UUID-11cbfd5c-b7fa-3d12-0f0c-3fdbbd2e4efb_Table_3.1).
I think we should follow it, tbh.

Reply:
Ya, fair enough.